packages/mechaml/mechaml.0.1/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+maintainer: "Yann Hamdaoui <yann.hamdaoui@gmail.com>"
+authors: "Yann Hamdaoui <yann.hamdaoui@gmail.com>"
+homepage: "https://github.com/yannham/mechaml"
+bug-reports: "https://github.com/yannham/mechaml/issues"
+license: "LGPL v3"
+dev-repo: "git://github.com/yannham/mechaml.git"
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: [make "install"]
+build-test: [
+  ["./configure" "--enable-tests"]
+  [make "test"]
+]
+build-doc: ["make doc"]
+remove: ["ocamlfind" "remove" "mechaml"]
+depends: [
+  "ocamlbuild" {build}
+  "ocamlfind" {build}
+  "cohttp"
+  "lwt"
+  "uri"
```

Comment:
Indeed, I forgot to include the _tags file generated by oasis. I updated the release snapshot at the same URL and it should be fine by now. Is there a way to restart the test suite without committing anything to the PR ?